recipes/hdmf_zarr/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "hdmf_zarr" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 281cfcd1581ed8e101953015179aedc01e55740980540f7b0e43f67110fa44f5
+  
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
It does not look like a compiler is needed to build this package, so you can remove this and the `build` section.